framework/db/Migration.php

```diff
@@ -552,9 +538,36 @@ public function dropCommentFromColumn($table, $column)
      */
     public function dropCommentFromTable($table)
     {
-        echo "    > drop comment from table $table ...";
-        $time = microtime(true);
+        $time = $this->beginCommand("drop comment from table $table");
         $this->db->createCommand()->dropCommentFromTable($table)->execute();
-        echo ' done (time: ' . sprintf('%.3f', microtime(true) - $time) . "s)\n";
+        $this->endCommand($time);
+    }
+
+    /**
+     * Prepares for a command to be executed, and outputs to the console
+     * 
+     * @param string $description the description for the command, to be output to the console.
+     * @return float the time before the command is executed, for the time elapsed to be calculated.
+     * @since 2.0.13
+     */
+    protected function beginCommand($description)
```

Comment:
there is no calculation in #14056 and it would be only moving an if statement to a separate method. In that case it was about reducing overhead. In this case its a calculation, which is subject to change so the risk of duplicate code is much higher.